### ISSUE TITLE
java.lang.UnsatisfiedLinkError: No implementation found

### ISSUE THREAD

**Issue by darkcoderrises** [servalproject/batphone#137]
```
java.lang.UnsatisfiedLinkError: No implementation found for int org.servalproject.servaldna.ServalDCommand.setInstancePath(java.lang.String) (tried Java_org_servalproject_servaldna_ServalDCommand_setInstancePath and Java_org_servalproject_servaldna_ServalDCommand_setInstancePath__Ljava_lang_String_2)
                      at org.servalproject.servaldna.ServalDCommand.setInstancePath(Native Method)
                      at org.servalproject.ServalBatPhoneApplication.onCreate(ServalBatPhoneApplication.java:324)
                      at android.app.Instrumentation.callApplicationOnCreate(Instrumentation.java:1119)
                      at android.app.ActivityThread.handleBindApplication(ActivityThread.java:5827)
                      at android.app.ActivityThread.-wrap1(Unknown Source:0)
                      at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1673)
                      at android.os.Handler.dispatchMessage(Handler.java:105)
                      at android.os.Looper.loop(Looper.java:172)
                      at android.app.ActivityThread.main(ActivityThread.java:6637)
                      at java.lang.reflect.Method.invoke(Native Method)
                      at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:240)
                      at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:767)
```
I am getting error the moment app opens up, can anyone help me with this?

**Comment by lakeman** [comment]
Should be fixed by 92099788

**Comment by darkcoderrises** [comment]
Yeah, it solves the problem. Thanks
Nice app btw, I was building a similar thing for a disaster recovery project, found this instead.
